### ISSUE TITLE
Ignore rar file releases on TorrentLeech

### ISSUE THREAD

**Issue by martimarkov** [Prowlarr/Indexers#346]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

The problem is that some releases are `rar`-ed. They show up first and later an unrar-ed release is available. 

### Describe the solution you'd like

The ability to ignore those results which have the tag `rar` in their list of tags. 

### Describe alternatives you've considered

None

### Anything else?

I'm not quite sure how to add conditions to filter the results from the JSON list of results. If someone points me in the direction or if we have something like that in another indexer - I'm happy to write the code for this 🙂

**Comment by martimarkov** [comment]
@mynameisbogdan I do feel that this could be seen as an improvement of Cardigann then. What would be the best place to do that?

Also we can do this filtering using selectors in HTML/CSS, maybe moving to something like JSONNET would allow for this

**Comment by martimarkov** [comment]
I meant more the fact that we can't do filtering in JSON as we can do it in HTML. If JSON is limited as a format maybe implementing another parser for the content would be an alternative. Hence why I mentioned jsonnet as it can contain simple logic. 

I'm also not sure where cardigann comes from (I only found out a go project which seems abandoned) and an implementation in Prowlarr but don't know what it's based on. 

Again I'm happy to write the code for this but not sure which implementation of cardigann to based it on

**Comment by martimarkov** [comment]
@mynameisbogdan Where can I find more info on V10? Is there a branch for v10 or is it just a planned feature for when v10 will be built?

Happy to contribute to the development. 🙂

**Comment by martimarkov** [comment]
@mynameisbogdan any timelines for V10? Anything you can share so I can help?

**Comment by martimarkov** [comment]
@mynameisbogdan LEGEND!!! ❤️

**Comment by ilike2burnthing** [comment]
- `Priscilla 2023` - 14 results
- `Priscilla 2023 -tags:rar` - 13 results
- `tt22041854` - 14 results
- `tt22041854 -tags:rar` - 0 results

The tracker itself doesn't support it.

**Comment by KptCheeseWhiz** [comment]
Yes, now I know, but it is not written anywhere. I """"fixed"""" (I think?) the issue by removing the `imdbid` in the `movie-search` mode from the definition. It would be nice to add a note next to the checkbox (or anywhere).

https://github.com/Prowlarr/Indexers/blob/08a2940ccaa03b94527e5719ca364a85ed979fbe/definitions/v7/torrentleech.yml#L71

**Comment by ilike2burnthing** [comment]
https://github.com/Jackett/Jackett/commit/f7ad39090c5429b5b4ea6a05d92d5d8785ab7fce
